app/javascript/packs/Dropdown.jsx

```diff
@@ -0,0 +1,20 @@
+import { h, render } from 'preact';
+import { Dropdown } from '../articles/components/Dropdown';
+
+function loadElement() {
+  //   const curatedLists = JSON.parse(
+  //     document.getElementById('curated-list').dataset.curatedLists,
+  //   );
+
+  const article = document.getElementById('articleID');
+  if (article) {
+    const articleObject = { id: JSON.parse(article.dataset.id) };
+    render(<Dropdown article={articleObject} />);
+  }
+}
+
+window.InstantClick.on('change', () => {
+  loadElement();
+});
+
+loadElement();
```

Comment:
For posterity: this pack was created to attempt to load the Dropdown component onto the social actions sidebar of the Article show page, but that feature is not yet complete.